modules/git/repo.go

```diff
@@ -283,7 +283,7 @@ type DivergeObject struct {
 // GetDivergingCommits returns the number of commits a targetBranch is ahead or behind a baseBranch
 func GetDivergingCommits(ctx context.Context, repoPath, baseBranch, targetBranch string) (do DivergeObject, err error) {
 	cmd := NewCommand(ctx, "rev-list", "--count", "--left-right").
-		AddDynamicArguments(baseBranch + "..." + targetBranch)
```

Comment:
:raised_eyebrow: How does this help?

Reply:
Doesn't directly help, but it's good to have:

```
2024/03/20 12:56:57 ...repository/branch.go:141:loadOneBranch() [E] CountDivergingCommits: exit status 128 - fatal: ambiguous argument 'main...refs/heads/tto-device-discovery': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
 - fatal: ambiguous argument 'main...refs/heads/tto-device-discovery': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```